api-datatypes.go

```diff
@@ -41,6 +41,7 @@ type ObjectInfo struct {
 	LastModified time.Time `json:"lastModified"` // Date and time the object was last modified.
 	Size         int64     `json:"size"`         // Size in bytes of the object.
 	ContentType  string    `json:"contentType"`  // A standard MIME type describing the format of the object data.
+	Expires      string    `json:"expires"`      // The date and time at which the object is no longer able to be cached.
```

Comment:
`time.Time` would be the native type of Expires

Reply:
fixed